lib/vault/encrypted_model.rb

```diff
@@ -59,10 +59,13 @@ def vault_attribute(attribute, options = {})
           serializer.define_singleton_method(:decode, &options[:decode])
         end
 
+        self.attribute attribute.to_s, ActiveRecord::Type::Value.new,
+          :default => nil
+
         # Getter
         define_method("#{attribute}") do
           self.__vault_load_attributes! unless @__vault_loaded
-          instance_variable_get("@#{attribute}")
+          super()
```

Comment:
```suggestion
          super
```

Reply:
Due to the fact we're actually passing _no_ arguments in here (rather than the original arguments), we need to use parentheses. I just tried removing them, and the build was [not happy](https://travis-ci.org/hashicorp/vault-rails/jobs/534362639) - perhaps I hit this problem myself originally, because I usually leave parentheses out for a normal `super` call.